source/extensions/filters/network/rbac/rbac_filter.cc

```diff
@@ -10,6 +10,8 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RBACFilter {
 
+static const std::string shadow_policy_id_field = "shadow_effective_policyID";
```

Comment:
Maybe worth to refactor with the one used in the HTTP RBAC filter to use a single constant.

Reply:
What would be a good place to have it?

Reply:
Maybe under `source/extensions/filters/common/rbac`

Reply:
Addressed in 8e969d0.